pkg/reader/mysql/reader.go

```diff
@@ -17,8 +17,8 @@ type storage struct {
 }
 
 // NewStorage ...
-func NewStorage(conn *sql.DB) reader.Reader {
-	return generic.NewSqlReader(&storage{conn})
+func NewStorage(conn *sql.DB, t time.Duration) reader.Reader {
```

Comment:
IMO `t` is too implicit and tells only aboit the type, but not the purpose of the parameter